### PR TITLE
created a basic react app

### PR DESCRIPTION
**Pull request by Hambalieu**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,69 @@
...
+    this.setState({
+      cityData: cityResults.data[0]
+    })} catch(error){
+      this.setState({
+        renderError: true,
+        errorMessage: `Error Occured: ${error.response.status}, ${error.response.data.error}`
+
+      })
+    }
+  }
```
Comment (RivaD2) [inline src/App.js]: - Good work here! 👍  We can clean up the indentation for code inside the try block

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,69 @@
...
+
+        <main>
+          <form onSubmit={this.handleSubmit}>
+            <label>Pick a City!
+              <input name="city" type="text" />
+            </label>
+            <button type="submit">Explore!</button>
+          </form>
+        </main>
+      </>
```
Comment (RivaD2) [inline src/App.js]: - We can bring in Bootstrap to help us display the Form (although lab doesn't say we have to use Bootstrap for form, really just to show the map) and any additional information we want to show like the city name, lat and longitude
-  We can use a Card or any other Bootstrap component that you think will work nicely here for the other data/the map. Then we will target what we stored in state for city information, in this case it is `cityData`. If we `console.log(cityData.display_name)`, what do we get? Same goes for the `lat` and `lon`. 👍

**Comment by ghost** [comment]
Nice job, good start and I think Riva covered most of it. Let me know if you run into issues!

File: src/App.js [hunk 3]
```diff
@@ -1,69 +1,80 @@
...
+                </Card.Body>
+                <Card.Footer>
+                  <p>LocationIQ API &copy;hjallow </p>
+                </Card.Footer>
+
+              </Card>
+
+            </article> }
         </main>
       </>
```
Comment (RobbMAlexander) [inline src/App.js:75]: React elements look good-- shouldn't take much to rework it for any other data you want to display later.

**Comment by RobbMAlexander** [comment]
Putting this comment here for now as I can't place it in your server Repo: Basic server vitals look like they're just about ready, and your Constructor looks prepared --- just have to get the API calls going and you should be on track to feed the groomed data to your front-end.

**Pull request merged**